qemu/tests/blockdev_mirror_to_rbd.py

```diff
@@ -0,0 +1,36 @@
+from provider.blockdev_mirror_wait import BlockdevMirrorWaitTest
+from provider.virt_storage.storage_admin import sp_admin
+
+
+class BlockdevMirrorRBDNode(BlockdevMirrorWaitTest):
+    """
+    Block mirror with filter-node-name set test
+    """
+
+    def target_disk_define_by_params(self, params, image_name):
+        pool_name = params.get("storage_pool")
+        params.setdefault("target_path", pool_name)
+        vol = sp_admin.volume_define_by_params(image_name, params)
+        return vol
```

Comment:
remove this function as we add the rbd backend, all the functions should keep the same

Reply:
Have updated, thanks zhenchao.